### ISSUE TITLE
Android 5.1.1 (Nexus) Audio Playback Causes Frame Skipping 

### ISSUE THREAD

**Issue by ermanhaskan** [cocos2d/cocos2d-x#12942]
Hello

WAV. files played by SimpleAudioEngine's playEffect() method is causing frame skipping from time to time on Android 5.1.1 on different Nexus devices (tested on Nexus 5 and 6). Sounds are also not played properly when they are played rapidly which works without any problems on any other device.

I am receiving strange audio logs each time an effect is played:

D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(3: compress-offload-playback) kvpairs: routing=0
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(3: compress-offload-playback) kvpairs: closing=true
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(3: compress-offload-playback) kvpairs: exiting=1
D/offload_visualizer(  355): thread exit
D/audio_hw_primary(  355): disable_audio_route: reset and update mixer path: compress-offload-playback speaker
E/audio_hw_primary(  355): offload_thread_loop: Compress handle is NULL
W/AudioFlinger(  355): moveEffectChain_l() effect chain for session 0 not on source thread 0xb59c4000
D/audio_hw_primary(  355): adev_set_parameters: enter: rotation=90
D/mot_vr_audio_hw(  355): adev_set_parameters: rotation=90
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
D/audio_hw_primary(  355): out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2

**Comment by ermanhaskan** [comment]
Tried out 3.7 and this issue still persists.

**Comment by dumganhar** [comment]
@ermanhaskan, if you're using `SimpleAudioEngine` which is implemented by java `SoundPool`, we could not fix this issue since it's definitely a bug of `Android`.
I refactored a lot code of `experimental::AudioEngine` in v3.14. And use `software decoding` in v3.15 (https://github.com/cocos2d/cocos2d-x/pull/17233). 
I think it will work without problems uses the `AudioEngine` in latest v3 branch.
Therefore, I closed this issue temporarily. If you still meet audio issues in AudioEngine, feel free to create a new issue. :)
